optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/gwtjackson/LocalTimeJsonDeserializer.java

```diff
@@ -0,0 +1,20 @@
+package org.optaplanner.openshift.employeerostering.gwtui.client.gwtjackson;
+
+import java.time.LocalTime;
+
+import com.github.nmorel.gwtjackson.client.JsonDeserializationContext;
+import com.github.nmorel.gwtjackson.client.JsonDeserializer;
+import com.github.nmorel.gwtjackson.client.JsonDeserializerParameters;
+import com.github.nmorel.gwtjackson.client.stream.JsonReader;
+
+public class LocalTimeJsonDeserializer extends JsonDeserializer<LocalTime> {
+
+    @Override
+    protected LocalTime doDeserialize(JsonReader reader, JsonDeserializationContext ctx, JsonDeserializerParameters params) {
+        String text = reader.nextString();
+        // TODO the super source of DateTimeFormatter is broken
+        //        return LocalDateTime.parse(text, DateTimeFormatter.ISO_LOCAL_DATE_TIME);
+        return LocalTime.parse(text);
```

Comment:
What happens if the client and server have a different timezone?
To test that, you can use `-Duser.timezone=Europe/Sofia` for the server side. Your browser will still your normal timezone.

If that causes bug, this a known bug, so let's add the reproducer info in the TODO and create a jira.

Reply:
Everything worked fine when I ran the server with `-Duser.timezone/Europe/Sofia`. But that bug seems VERY odd. `LocalTime` should NOT have timezone information (`LocalTime` is used for `ShiftTemplate`; `OffsetTime` is used for both `EmployeeAvaliability` and `Shift`).

Reply:
Thanks for testing that - good to confirm that no such bug exists.